src/Polly.Core/Simmy/Fault/FaultChaosStrategy.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Polly.Telemetry;
+
+namespace Polly.Simmy.Fault;
```

Comment:
(and any other place VS might have created new files like this)

```suggestion
namespace Polly.Simmy.Fault;

```